components/automate-cli/pkg/verifyserver/server/api/v1/fqdn.go

```diff
@@ -37,6 +37,6 @@ func (h *Handler) CheckFqdn(c *fiber.Ctx) error {
 		return fiber.NewError(http.StatusBadRequest, "node_type should be automate or chef-infra-server, Please provide node_type.")
 	}
 
-	res := h.FqdnService.CheckFqdnReachability(*req, constants.DEFAULT_HTTPS_PORT, time.Minute*1)
+	res := h.FqdnService.CheckFqdnReachability(*req, constants.DEFAULT_HTTPS_PORT, time.Second*30)
```

Comment:
Are you sure on this, this impacts the node reachability check

Reply:
The API timeout was set to 35 seconds, and checking for reachablity for 60 seconds, even if the FQDN was reachable after 35 second, the API request might have ended already. So based on that I have reduced it to 30 sec.